app/connectors/ApmConnector.scala

```diff
@@ -62,6 +63,10 @@ class ApmConnector @Inject() (http: HttpClient, config: ApmConnector.Config, met
     http.GET[Map[ApiContext, ApiData]](s"${config.serviceBaseUrl}/api-definitions?applicationId=${applicationId.value}")
   }
 
+  def fetchAllOpenAccessApis(environment: Environment)(implicit hc: HeaderCarrier): Future[Map[ApiContext, ApiData]] = {
+    http.GET[Map[ApiContext, ApiData]](s"${config.serviceBaseUrl}/api-definitions/open?environment=${environment.toString}")
```

Comment:
Instead of hand cracking URI params into the URI string, I am sure you can pass them into http.GET as a parameter after the URI string.

Reply:
Nicely reminded